src/plugins/plugin.h

```diff
@@ -283,6 +283,10 @@ struct janus_plugin {
 	 * @param[out] error An integer that may contain information about any error */
 	void (* const create_session)(janus_plugin_session *handle, int *error);
 	/*! \brief Method to handle an incoming message/request from a peer
+	 * @note The Janus core increases the references to both the \c message and \c jsep
```

Comment:
I think that, since in this case this is a method that goes from core to plugin, it's more correct to say something like "The Janus core leaves ownership of both ...... objects to plugins. Ths means..". The "increases" part makes more sense for `push_event` since in that case the core does indeed increase the references to those objects, since plugins may decrease their own.